### PR TITLE
Increase timeouts for identified erc20 tests

### PR DESCRIPTION
**Pull request by david-zk**
Tests fail now with smaller timeout

@dartdart26 @immortal-tofu @youben11

### CODE REVIEW

**Comment by immortal-tofu** [comment]
You can define default timeout in `hardhat.config.ts`. Maybe it's better?

File: test/governor/GovernorZama.ts [hunk 1]
```diff
@@ -35,7 +33,7 @@ describe('GovernorZama', function () {
 
   beforeEach(async function () {
     // Increase timeout for beforeEach
-    this.timeout(timeout);
+    this.timeout(180000);
```
Comment (dartdart26) [inline test/governor/GovernorZama.ts]: Do we need to set to 180000 here?

**Review by dartdart26 — APPROVED** [review]

File: test/governor/GovernorZama.ts (hunk 1 above)
Comment (david-zk) [reply test/governor/GovernorZama.ts]: Not sure, will try rerunning tests without

Comment (david-zk) [reply test/governor/GovernorZama.ts]: It passes, we can merge 👍

**Pull request merged**